rcsb/utils/insilico3d/ModelReorganizer.py

```diff
@@ -287,6 +306,58 @@ def __rebuildEntryIds(self, dataContainer, sourceModelEntryId, sourceModelDb, in
 
         return dataContainer
 
+    def __rebuildDateDetails(self, dataContainer, sourceModelEntryId, sourceArchiveReleaseDate, sourceArchiveModifiedDate):
+        """Add or rebuild release and revision date details for the dataContainer.
+
+        Mainly for ModelArchive models which currently lack this information in the mmCIF file (as of 19-Oct-2022 dwp).
+
+        Args:
+            dataContainer (object): mmcif.api.DataContainer object instance
+            sourceModelEntryId (str): source entry ID
+            sourceArchiveReleaseDate (str): release date for dataContainer, obtained from source model website (e.g., '2022-09-28')
+            sourceArchiveModifiedDate (str): last-modified date for dataContainer, obtained from source model website (e.g., '2022-09-28T17:22:53.310750Z')
+
+        Returns:
+            dataContainer: updated dataContainer object
+        """
+        # If pdbx_database_status.* doesn't exist, create it and add release date
+        if not dataContainer.exists("pdbx_database_status"):
+            dataContainer.append(
+                DataCategory(
+                    "pdbx_database_status",
+                    attributeNameList=["entry_id", "recvd_initial_deposition_date", "status_code"],
+                    rowList=[[sourceModelEntryId, sourceArchiveReleaseDate, "REL"]]
+                )
+            )
+
+        minorRevision = "0"
+        revisionHistoryOrdinal = None
+
+        # If _pdbx_audit_revision_history.* doesn't exist, create it and add modified date
+        if not dataContainer.exists("pdbx_audit_revision_history"):
+            if sourceArchiveModifiedDate[0:10] != sourceArchiveReleaseDate[0:10]:
+                minorRevision = "1"
```

Comment:
I think we should add the bare minimum information required for RCSB.org and limit this to the initial release of the model with major revision 1 and minor revision 0.

Reply:
Sure understand, will do. Thanks.